README.md

```diff
@@ -85,7 +85,7 @@ step-by-step instructions.<br/>
 
 Excited to Dive In? 💡 <br>
 Start building with Taipy today! Our
-[Getting Started Guide]https://docs.taipy.io/en/develop/tutorials/getting_started/)
+[Getting Started Guide](https://docs.taipy.io/en/develop/tutorials/getting_started/)
```

Comment:
This as already been solved in another PR. You can revert this and/or rebase your branch.